Balloon/app/device.cpp

```diff
@@ -214,18 +233,20 @@ PTCHAR CDevice::GetDevicePath(IN LPGUID InterfaceGuid)
             HeapFree(GetProcessHeap(), 0, DeviceInterfaceList);
         }
 
-        DeviceInterfaceList = (PWSTR)HeapAlloc(GetProcessHeap(),
-            HEAP_ZERO_MEMORY, DeviceInterfaceListLength * sizeof(WCHAR));
+        DeviceInterfaceList = (PWSTR)HeapAlloc(
+                                 GetProcessHeap(), HEAP_ZERO_MEMORY, DeviceInterfaceListLength * sizeof(WCHAR));
```

Comment:
Can we not skip the line in this case? If the line too long, the parameters should be slip with the usual rule of parameter per line